crates/aranya-quic-channels/tests/test.rs

```diff
@@ -0,0 +1,137 @@
+use std::{net::SocketAddr, ops::DerefMut, sync::Arc, time::Duration};
+
+use anyhow::Result;
+use aranya_crypto::csprng::rand;
+use aranya_quic_channels::{run_channels, AqcClient};
+use aranya_runtime::{
+    protocol::{TestActions, TestEngine, TestSink},
+    storage::memory::MemStorageProvider,
+    ClientState,
+};
+use bytes::Bytes;
+use s2n_quic::{provider::congestion_controller::Bbr, Server};
+use tokio::sync::Mutex as TMutex;
+
+#[test_log::test(tokio::test)]
+async fn test_channels() -> Result<()> {
```

Comment:
To differentiate from other tests (e.g. afc), we should include aqc in the name of the test:
```suggestion
async fn test_aqc_channels() -> Result<()> {
```
or
```suggestion
async fn test_quic_channels() -> Result<()> {
```